CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.0.0
```

Comment:
I know it is a breaking change and we should update the major version, but let's just update the minor version. I don't think we need to make a major release for just that small change. Package users will be able to fix that quickly.